### PR TITLE
ci: Package RACK CLI and requirements as wheels

### LINKED ISSUE #198: RACK-box: Install RACK CLI outside of a virtualenv
We currently install the RACK CLI into a virtualenv, then use `xargs`+`perl` to fix up some paths so that we can actually run it inside the RACK-box Docker image. This is a bit indirect: We should instead create a binary distribution (like a wheel file) that we install using standard methods inside the RACK-box image. This will have the following advantages:

- Save us some cleverness and effort in our CI pipeline
- Make it easier to use the RACK CLI in the RACK-box image (by installing it globally, rather than in a virtualenv)
- Ensure that we have the CLI properly configured so that it can be packaged in a binary distribution

### PR DESCRIPTION
**Pull request by langston-barrett**
Fixes #198.

### CODE REVIEW

File: .github/workflows/rack-box-continuous.yml [hunk 1]
```diff
@@ -77,15 +95,16 @@ jobs:
...
+        pip3 wheel --wheel-dir=wheels -r requirements.txt
+        pip3 wheel --wheel-dir=wheels .
         cd ${{ github.workspace }}
-        tar cfz RACK/rack-box/files/rack.tar.gz --exclude=.git --exclude=.github --exclude=rack-box --exclude=tools RACK
+        tar cfz RACK/rack-box/files/rack-cli.tar.gz RACK/RACK-Ontology/cli/{setup-rack.sh,wheels}
+
+    - name: Package RACK ontology and data
+      if: steps.cache-rack-ontology.outputs.cache-hit != 'true'
+      run: |
+        tar cfz RACK/rack-box/files/rack.tar.gz --exclude=.git --exclude=.github --exclude=RACK-Ontology/cli --exclude=rack-box --exclude=tools RACK
```
[the remark this answers is not in the document]
Comment (kquick) [reply .github/workflows/rack-box-continuous.yml:107]: Are we packaging assist elsewhere?  (The assist tool is not part of the CLI wheel.)

File: .github/workflows/rack-box-release.yml [hunk 2]
```diff
@@ -41,9 +41,27 @@ jobs:
     - name: Cache RACK CLI
       uses: actions/cache@v2
       id: cache-rack-cli
+      with:
```
[the remark this answers is not in the document]
Comment (kquick) [reply .github/workflows/rack-box-release.yml:44]: That sounds like it's more work than is needed, so the current setup is probably best.

File: .github/workflows/rack-box-continuous.yml (hunk 1 above)
[the remark this answers is not in the document]
Comment (langston-barrett) [reply .github/workflows/rack-box-continuous.yml]: @kquick No, we are not packaging the assist tooling yet. Happy to do it if you think we should though (let's open an issue if so)!

[the remark this answers is not in the document]
Comment (langston-barrett) [reply .github/workflows/rack-box-continuous.yml]: > Now that both assist and cli are in top-level directories, the tar will need to exclude assist too. Will need to change similar line in rack-box-release.yml too.

Done, thanks for the note.

**Pull request merged**